### DISCUSSION TITLE
PaddleX/3.5/pipeline_deploy/serving

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleX#5125] [General]
# PaddleX/3.5/pipeline_deploy/serving

https://paddlepaddle.github.io/PaddleX/3.5/pipeline_deploy/serving.html

**Comment by FrostbladeDev** [comment]
请问下 服务化部署 ，SDK paddlex_hps_PaddleOCR-VL-1.5_sdk 启动成功，解析图片接口 （/v2/models/layout-parsing/infer）报错是啥原因呢？


服务端日志
I0429 01:06:50.037227 7 grpc_server.cc:4117] Started GRPCInferenceService at 0.0.0.0:8001
I0429 01:06:50.038592 7 http_server.cc:2815] Started HTTPService at 0.0.0.0:8000
I0429 01:06:50.080724 7 http_server.cc:167] Started Metrics Service at 0.0.0.0:8002
[    INFO] [2026-04-29 01:10:08,013] [5798dbd200984b3c9e4c93455036cce1] [0a200f5180e34b8a837565d729bec7ac] - Received batch of size 1
[    INFO] [2026-04-29 01:10:08,135] [5798dbd200984b3c9e4c93455036cce1] [0a200f5180e34b8a837565d729bec7ac] - Request 759dc40c-e31f-413d-83d4-bba1278b412e received
/paddlex/py310/lib/python3.10/site-packages/paddle/tensor/creation.py:1088: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach(), rather than paddle.to_tensor(sourceTensor).
  return tensor(
/paddlex/py310/lib/python3.10/site-packages/paddle/utils/decorator_utils.py:420: Warning:
Non compatible API. Please refer to https://www.paddlepaddle.org.cn/documentation/docs/en/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.max.html first.
  warnings.warn(
